campaign/snowplow.js

```diff
@@ -139,5 +147,12 @@ module.exports = {
   flush: () => {
     /* istanbul ignore next */
```

Comment:
You can remove this as it looks like the specs are actually covering calling emitter.flush();

Reply:
Unfortunately, removing it makes it say its uncovered. Maybe its because I'm calling `emitter.flush()` directly instead of through the `flush()` function.